libretroshare/src/util/rsnet.h

```diff
@@ -84,6 +84,11 @@ bool isExternalNet(const struct in_addr *addr);
 // uses a re-entrant version of gethostbyname
 bool rsGetHostByName(const std::string& hostname, in_addr& returned_addr) ;
 
+// Get hostName address using specific DNS server
+// Using it allow to direct ask our Address to IP, so no need to have a DNS (IPv4 or IPv6 ???).
+// If we ask to a IPv6 DNS Server, it respond for our IPv6 address.
+bool rsGetHostByNameSpecDNS(const std::string& servername, const std::string& hostname, std::string& returned_addr, int timeout_s = -1);
```

Comment:
Please add documentation in doxygen format, also for the timeout_s paramether

Reply:
Done